File: app/src/main/java/com/silentgames/silent_planet/model/entities/ground/Player.java
package com.silentgames.silent_planet.model.entities.ground;

import com.silentgames.silent_planet.model.entities.EntityTypeEx;

/**
 * Created by gidroshvandel on 10.07.16.
 */
public class Player extends EntityTypeEx {

    private int crystals = 0;

    private String playerName;

    public int getCrystals() {
        return crystals;
    }

    public void setCrystals(int crystals) {
        this.crystals = crystals;
    }

    public String getPlayerName() {
        return playerName;
    }

    public void setPlayerName(String playerName) {
        this.playerName = playerName;
    }

}

File: app/src/main/java/com/silentgames/silent_planet/model/entities/ground/Player.kt
package com.silentgames.silent_planet.model.entities.ground

import android.content.Context
import com.silentgames.silent_planet.model.entities.EntityType
import com.silentgames.silent_planet.model.fractions.Fractions

/**
 * Created by gidroshvandel on 10.07.16.
 */
abstract class Player(
        context: Context,
        fraction: Fractions,
        override var isCanMove: Boolean = true,
        override var isCanFly: Boolean = false
) : EntityType(context, fraction)
